pkg/internal/server/grpc/server_test.go

```diff
@@ -43,9 +45,9 @@ func TestNewServer(t *testing.T) {
 	}
 
 	t.Run("tls-load-error", func(t *testing.T) {
-		th := runtime.NewTestHarness().WithTestTLSFiles(t)
-		defer th.RemoveTestTLSFiles(t)
-		rta := th.RuntimeArgs()
+		rth := runtime.NewTestHarness().WithTestTLSFiles(t)
```

Comment:
Using `rth` for the runtime test harness as opposed to `th` for the local package test harness.